content/posts/2019-05-02-apollo-client-in-practice.md

```diff
@@ -96,8 +110,11 @@ to an array rows of cells
 # Next steps
 
 Apollo is excellent for merging data from multiple sources (in this case GraphQL, REST, local state and cache) and 
-functions as a "single source of truth" which should solve the state synchronization problems. These are some other 
-things you might want to take into account when working with Apollo Client.
+functions as a "single source of truth" which should solve the state synchronization problems. The local fields that
+Apollo uses in its local state manager can derive data, moving expensive operations from component render functions
+to resolvers in its application level cache. Although the issues mentioned in the introduction are now dealt with, we did 
+encounter plenty of other issues I may dive into later. However, these are some things that you might want to 
+take into account when working with Apollo Client.
```

Comment:
`.` should be a `:` (I'm fine with the following listing not being formatted as a list.)

Reply:
I wondered about this before submitting, but I modified it.